mpire/dashboard/static/refresh.js

```diff
@@ -13,9 +13,9 @@ $(function() {
 
 var progress_bar_animation_duration = 450;
 var refresh_interval = 500;
+var last_refresh = 1;
 var completed_pb_ids = {};
-refresh();
-setInterval(refresh, refresh_interval);
+var interval_id = setInterval(refresh, refresh_interval);
```

Comment:
For some reason calling `refresh()` before setting the interval causes finished progress-bars not to show up with the latest commit (neither in firefox nor chrome). But leaving it out causes a half second delay.